plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt

```diff
@@ -143,6 +143,11 @@ open class HtmlRenderer(
             }
         }
 
+    fun FlowContent.withHtml(content: String): Unit = when (this){
```

Comment:
Why do we need it? It is not used anywhere

Reply:
We do need this in scala3doc. Also it will be super usefull in any other non-Kotlin usages.